hydra-main/src/main/java/com/addthis/hydra/job/web/resources/JobsResource.java

```diff
@@ -1016,6 +1019,8 @@ public Response startJob(@QueryParam("jobid") Optional<String> jobIds,
                 IJob job = spawn.getJob(jobId);
```

Comment:
This block of code of validating and kicking a job is identical to the code above. They should be refactored into a method to be used in both places